core/src/stored/device_resource.h

```diff
@@ -28,6 +28,8 @@
 #include "stored/autoxflate.h"
 #include "lib/bareos_resource.h"
 
+#include <optional>
+
```

Comment:
There's no use of optional in this header
```suggestion
```

Reply:
done